hypha/templates/base-apply.html

```diff
@@ -1,102 +1,62 @@
-        <script>
-            window.addEventListener('DOMContentLoaded', (event) => {
-                document.addEventListener("htmx:beforeRequest", function() { NProgress.start(); });
-                document.addEventListener("htmx:afterRequest", function() { NProgress.done(); });
-            });
-            {% comment %}
-            If the htmx response is either 403, 404, or 500, display the error page.
-            Please note that the error replaces the entire page, rather than displaying
-            the error response in the target element.
-            https://stackoverflow.com/a/74823597/782901
-            {% endcomment %}
-            document.addEventListener("htmx:beforeOnLoad", function (event) {
-                const xhr = event.detail.xhr
-                if (xhr.status == 500 || xhr.status == 403 || xhr.status == 404) {
-                    event.stopPropagation() // Tell htmx not to process these requests
-                    document.children[0].innerHTML = xhr.response // Swap in body of response instead
-                }
-            });
-        </script>
-        <!-- htmx end -->
-
-        <script src="{% static 'js/jquery.min.js' %}"></script>
-        <script src="{% static 'js/js.cookie.min.js' %}"></script>
-
-        <!-- alpine js start -->
-        <style> [x-cloak] {display: none !important} </style>
-        <!-- Alpine Plugins: installed with npm -->
-        <script defer src="{% static 'js/apply/vendor/alpine-focus.min.js' %}"></script>
-
-        <!-- Alpine Core: installed with npm  -->
-        <script defer src="{% static 'js/apply/vendor/alpine.min.js' %}"></script>
-        <!-- alpine.js end -->
-
-        <!-- modules -->
-        <script type="module" src="{% static 'js/esm/github-relative-time-element-4-3-0.js' %}"></script>
-        <script type="module" src="{% static 'js/esm/github-filter-input-element-0-1-1.js' %}"></script>
-    </head>
-
-    <body
-        hx-ext="multi-swap"
-        hx-headers='{"X-CSRFToken": "{{ csrf_token }}"}'
-        class="antialiased {% block body_class %}template-{{ page.get_verbose_name|slugify }}{% endblock %}"
-    >
-        {% include "includes/sprites.html" %}
-
-        {% if messages %}
-            {% include 'includes/messages.html' %}
-        {% endif %}
-
-        <header class="header">
-
-            <div class="header__inner wrapper wrapper--large">
-                <a href="{{ settings.utils.SystemMessagesSettings.site_logo_link|default:"/" }}" aria-label="Home link">
-                    {% if settings.utils.SystemMessagesSettings.site_logo_default %}
-                        {% image settings.utils.SystemMessagesSettings.site_logo_default width-215 as logo_default %}
-                        <img class="header__logo header__logo--desktop"
-                             width="215"
+{% extends "base.html" %}
+{% load i18n static wagtailcore_tags heroicons %}<!doctype html>
```

Comment:
Need to add `wagtailimages_tags` here.

Reply:
Since this page doesn't use any `{% image ..}` tag `wagtailimages_tags` is not needed on this page. Are you getting any error?